src/views/Home/Home.scss

```diff
@@ -1,18 +1,17 @@
+@use "~components/Button/Button";
+
```

Comment:
This shouldn't be necessary if we import the scss file inside `components/Button/index.view.tsx`. If it's not there, we should import it in that file and remove this line.

Reply:
yep, turned out this wasn't necessary since the index for `buttons` was already packaged with `buttons.scss` & included in the `Home` index.

Reply:
Ahh gotcha! I didn't see line 13. That's on me!